EKF/tests/pytest/lint

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+###############################################################################
```

Comment:
AFAIK, no .py file extension is quite standard for executable Python scripts (see, e.g., the `pip` command).